runtime/doc/api.txt

```diff
@@ -1884,7 +1885,7 @@ nvim_parse_cmd({str}, {opts})                               *nvim_parse_cmd()*
           • verbose: (integer) |:verbose|. -1 when omitted.
           • vertical: (boolean) |:vertical|.
           • split: (string) Split modifier string, is an empty string when
-            there's no split modifier. If there is a split modifier it can be
+            there's no split modifier. If there is a split modifier, it can be
```

Comment:
Is the grammar tool on "German mode"? Commas are not so commonly needed in English, or at least US English.

Reply:
It's certainly not in "API reference mode". You should distinguish the parts that are (concise) prose, e.g., in top-level module descriptions and examples, and those that are simply lists for reference. The latter needs to be as terse as possible for layout reasons (good layout is also important for good documentation!), while the former have more room for grammatical changes _if_ they make the text less ambiguous.

Reply:
Both ways are correct, but the docs are currently written in "comma free" style and it's a stylistic choice.

Reply:
If needed, I can remove the commas I added, but I feel like it just adds clarity.